studio/tests/pages/projects/logs-query.test.js

```diff
@@ -139,17 +141,52 @@ test('custom sql querying', async () => {
 
 test('datepicker interaction updates query params', async () => {
   render(<LogsExplorerPage />)
-  clickDropdown(await screen.findByText(/Last day/))
+  clickDropdown(await screen.findByText(/Last 24 hours/))
   userEvent.click(await screen.findByText(/Last 3 days/))
 
-  const router = useRouter()
-  expect(router.push).toBeCalledWith(
-    expect.objectContaining({
-      query: expect.objectContaining({
-        its: expect.any(String),
-      }),
-    })
-  )
+  useProjectSubscription.mockReturnValue({
```

Comment:
this should be moved to before the interaction, otherwise we would be simulating events as a free user, so the test will always pass

Reply:
what we can do, is to have 1 inetraction to assert the popup on the  free user, then replace the mock to simulate a pro user. 
free -> would not change router params, would show popup
pro -> would change router params, would not show popup